client/usr/lib/systemd/system/prometheus-tunnel.service

```diff
@@ -3,6 +3,7 @@ Description=Open SSH tunnel to prometheus server with reverse port
 After=network.target
 
 [Service]
+EnvironmentFile=/usr/lib/systemd/system/prometheus-tunnel.service.d/tunnel.conf
```

Comment:
This isn't required. See the section on `service.d` in https://www.freedesktop.org/software/systemd/man/systemd.unit.html